linkatos/message.py

```diff
@@ -1,7 +1,11 @@
 import re
+import os
```

Comment:
Looks like everything belongs to the `message` module which means to me, it's not a good module. We need to find a way to separate better the code in files. (same applies for tests).